RecoLocalCalo/HGCalRecProducers/interface/HGCalLayerTiles.h

```diff
@@ -66,11 +60,9 @@ class HGCalLayerTilesT {
   }
 
   int getPhiBin(float phi) const {
```

Comment:
The method `getPhiBin` becomes now identical to `TICLLayerTile::phiBin`

Analogously, `getEtaBin` in this class is the same as `TICLLayerTile::etaBin,` and the same for `getGlobalBinByBin` and `getGlobalBinEtaPhi`.

(Moreovers, it is recommended to not start getter methods with "get": as such the names of the methods as in TICLLayerTile must be preferred.

Probably taking them from a common class could help avoid having to fix any of them independently from the others